app/models/comment.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class Comment < ApplicationRecord
+  validates :user_id, presence: true
```

Comment:
- [ ] `belongs_to` association already adds presence validation. So `belongs_to :user`, then `user_id` is required by default. Same for `post_id`.